### PR TITLE
Rename config properties so that they're similarly named as in other Splunk distros

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
e.g. splunk-otel-python, splunk-otel-js

README will be updated after release

### CODE REVIEW

**Comment by jkwatson** [comment]
Since this will break existing users, should we support both for at least a little while?

**Comment by mateuszrzeszutek** [comment]
> Since this will break existing users, should we support both for at least a little while?

Good point - I'll do that for the auth token, since it seems to be the only one that's actually used by our users. Thanks!

**Comment by jkwatson** [comment]
> > Since this will break existing users, should we support both for at least a little while?
> 
> Good point - I'll do that for the auth token, since it seems to be the only one that's actually used by our users. Thanks!

It might be interesting to log a warning about the deprecated name at startup time if we see it being used.

File: custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java [hunk 1]
```diff
@@ -44,6 +43,12 @@ public SpanExporter createExporter(ConfigProperties config) {
     String token = config.getString(SPLUNK_ACCESS_TOKEN);
     if (token == null) {
       token = config.getString(SPLUNK_ACCESS_TOKEN_OLD);
+      if (token != null) {
+        log.warn(
+            "Deprecated property '{}' was set; consider using '{}' instead",
```
Comment (jkwatson) [inline custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java:48]:
```suggestion
            "Deprecated property '{}' was set; please use '{}' instead",
```

File: custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java [hunk 2]
```diff
@@ -31,15 +31,25 @@
 public class JaegerThriftSpanExporterFactory implements ConfigurableSpanExporterProvider {
   private static final Logger log = LoggerFactory.getLogger(JaegerThriftSpanExporterFactory.class);
 
-  static final String SIGNALFX_AUTH_TOKEN = "signalfx.auth.token";
+  @Deprecated static final String SPLUNK_ACCESS_TOKEN_OLD = "signalfx.auth.token";
+  static final String SPLUNK_ACCESS_TOKEN = "splk.access.token";
```
Comment (iNikem) [inline custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java:35]: Why the heck did we agree on `splk` and not `splunk` ???

File: instrumentation/netty-3.8/build.gradle [hunk 3]
```diff
@@ -15,5 +15,5 @@ dependencies {
 }
 
 tasks.withType(Test) {
-    jvmArgs '-Dsplunk.context.server-timing.enabled=true'
+    jvmArgs '-Dsplk.context.server-timing.enabled=true'
```
Comment (iNikem) [inline instrumentation/netty-3.8/build.gradle:18]: 😭 Why??

File: custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java (hunk 2 above)
Comment (mateuszrzeszutek) [reply custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java:35]: I don't know... there's even a conversation on slack about using `SPLK_` and we both left 👍  there 😭 🤣   nobody knows why now 🤷‍♂️

File: instrumentation/netty-3.8/build.gradle (hunk 3 above)
Comment (breedx-splk) [reply instrumentation/netty-3.8/build.gradle:18]: I wasn't around for this, but it _is_ the ticker symbol and it _is_ shorter.  😁 It's just unpronounceable.... 🤮

**Comment by iNikem** [comment]
We have agreed on `SPLUNK_` prefix.

**Review by iNikem — APPROVED** [review]

**Pull request merged**